modules/sno-du-configuring-access-to-the-bare-metal-server.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-access-to-the-bare-metal-server_{context}"]
+= Configuring access to the bare metal server
+
+This configuration provides the location of the installation image.
```

Comment:
This configuration provides access and specific configuration of the bare metal server.

Reply:
change made